docs/content.zh/release-notes/flink-1.13.md

```diff
@@ -0,0 +1,227 @@
+
+While recovering from unaligned checkpoints, users can now change the parallelism of the job. This
+change allows users to quickly upscale the job under backpressure.
+
+### SQL
+
+#### Officially deprecate the legacy planner
+
+##### [FLINK-21709](https://issues.apache.org/jira/browse/FLINK-21709)
+
+The old planner of the Table & SQL API is deprecated and will be dropped in Flink 1.14. This means
+that both the BatchTableEnvironment and DataSet API interop are reaching end of life. Use the
+unified TableEnvironment for batch and stream processing with the new planner, or the DataStream API
+in batch execution mode.
+
+#### Disable problematic cast conversion between NUMERIC type and TIMESTAMP type
+
+##### [FLINK-21698](https://issues.apache.org/jira/browse/FLINK-21698)
+
+The CAST operation between `NUMERIC` type and `TIMESTAMP` type is problematic and is disabled now,
+e.g.
+`CAST(numeric AS TIMESTAMP(3))` is disabled and should use `TO_TIMESTAMP(FROM_UNIXTIME(numeric))`
+instead.
+
+#### Support USE MODULES syntax
+
+##### [FLINK-21298](https://issues.apache.org/jira/browse/FLINK-21298)
+
+The term MODULES is a reserved keyword now. Use backticks to escape column names and other
+identifiers with this name.
+
+#### Update TableResult.collect()/TableResult.print() to the new type system
+
+##### [FLINK-20613](https://issues.apache.org/jira/browse/FLINK-20613)
+
+`Table.execute().collect()` might return slightly different results for column types and row kind.
+
+#### Update the Row.toString method
+
+##### [FLINK-18090](https://issues.apache.org/jira/browse/FLINK-18090)
+
+The `Row.toSting()` method has been reworked. This is an incompatible change. If the legacy
+representation is still required for tests, the old behavior can be restored via the flag
+`RowUtils.USE_LEGACY_TO_STRING` for the local JVM. However, relying on the row's string
+representation for tests is not a good idea in general as field data types are not verified.
+
+### Runtime
+
+#### BoundedOneInput.endInput is called when taking synchronous savepoint
+
+##### [FLINK-21132](https://issues.apache.org/jira/browse/FLINK-21132)
+
+`endInput()` is not called anymore (on BoundedOneInput and BoundedMultiInput) when the job is
+stopping with savepoint.
+
+#### Remove JobManagerOptions.SCHEDULING_STRATEGY
+
+##### [FLINK-20591](https://issues.apache.org/jira/browse/FLINK-20591)
+
+The configuration parameter `jobmanager.scheduler.scheduling-strategy` has been removed, because
+the `legacy` scheduler has been removed from Flink 1.13.0.
+
+#### Warn user if System.exit() is called in user code
+
+##### [FLINK-15156](https://issues.apache.org/jira/browse/FLINK-15156)
+
+A new configuration value `cluster.intercept-user-system-exit` allows to log a warning, or throw an
+exception if user code calls `System.exit()`.
+
+This feature is not covering all locations in Flink where user code is executed. It just adds the
+infrastructure for such an interception.
```

Comment:
There is no exhaustive list of locations which are not covered. Somebody needs to carefully look into this and make sure that all cases are covered.
This work is tracked in https://issues.apache.org/jira/browse/FLINK-21307. I propose we just link the ticket here in the release notes?

```suggestion
This feature is not covering all locations in Flink where user code is executed. It just adds the
infrastructure for such an interception. We are tracking this improvement in [FLINK-21307](https://issues.apache.org/jira/browse/FLINK-21307)
```

I'll ping the AWS guys again to complete this feature.